src/components/OurPartners.js

```diff
@@ -15,25 +15,45 @@ import Fisheries from '../assets/fisheries.png';
 const styles = theme => ({
   parentContainer: {
     padding: '4.0625rem 0 5.0625rem',
-    [theme.breakpoints.up('md')]: {
+    [theme.breakpoints.up('lg')]: {
       padding: '120px 120px 140px 124px'
     }
   },
   cardInfo: {
-    width: '240px',
-    height: '360px',
+    height: '19.5rem',
     backgroundColor: '#ffffff',
     boxShadow: '0 30px 60px 0 rgba(0, 0, 0, 0.05)',
     flex: '1',
     textAlign: 'center',
-    margin: '10px',
+    margin: '1rem',
+    transition: 'transform 1s',
     '&:hover': {
       transform: 'scale(1.1)'
     },
-    [theme.breakpoints.up('md')]: {
-      width: '400px',
-      height: '400px',
-      margin: '10px 40px'
+    [theme.breakpoints.up('lg')]: {
+      width: '23.4375rem',
+      height: '23.4375rem',
+      margin: '1.5rem'
+    },
+    [theme.breakpoints.width('1024')]: {
```

Comment:
Use breakpoints: `xs`, `sm`, `md`, etc.

Reply:
And these properties here are **exactly** like `lg` above. Why a different `breakpoint` then?

Reply:
@kilemensi Removed the `breakpoint 1024`. It wasn't really necessary at this point. Instead, moved the breakpoint from `md` to the `lg` . This fixed the issue on the ` Ipad pros ` and all other medium devices.